Guevados/jamMetroidvania
# Feature/pj running animations

# Problem

As PJ I want to be animated on movements

# Solution 

Handle animations for PJ when move left/right

# How to test

Run game and check left right movement animation pressing keys A or D


https://github.com/Guevados/jamMetroidvania/assets/28446213/3735323d-2778-4d43-8137-b4af06fa924d
